### PR TITLE
handle not running vms

### PR DESCRIPTION
**Pull request by pkliczewski**
There are several phases that vms can be in. Only in Running phase there is hostName available. This patch improves how we handle other phases.

There are potential issues with cert verification when using self signed ca. In this PR we add new param to connect without cert verification and host validation.

Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

### CODE REVIEW

**Comment by pkliczewski** [comment]
@wottop  @dankenigsberg please take a look

File: virtwho/virt/kubevirt/kubevirt.py [hunk 1]
```diff
@@ -136,12 +141,12 @@ def getHostGuestMapping(self):
 
         for vm in vms['items']:
             spec = vm['spec']
-            host_name = vm['status']['nodeName']
 
             # a vm is not scheduled on any hosts
-            if host_name is None:
+            if vm['status']['phase'] != 'Running':
```
Comment (dankenigsberg) [inline virtwho/virt/kubevirt/kubevirt.py]: Frankly, virt-who does not really care if the VM is `Running` it only cares if it is mapped to a host. Thus I think that 
```
host_name = vm['status'].get('nodeName')
```
would be a more precise change.

**Review by dankenigsberg — CHANGES_REQUESTED** [review]
Thanks for fixing these and adding the tests!

I'd have two different commits for the two different issues, but it's not my project to enforce.

File: virtwho/virt/kubevirt/kubevirt.py (hunk 1 above)
Comment (pkliczewski) [reply virtwho/virt/kubevirt/kubevirt.py]: As far as I understand how k8s work when a vm is `Running` phase `nodeName` always will be there. Do you suggest to make the changes as minimal as possible?

Comment (dankenigsberg) [reply virtwho/virt/kubevirt/kubevirt.py]: No, my issue is of precision. If for some reason KubeVirt has a bug and does not report the node on a Running VM. "My" suggested code just skips it. If KubeVirt introduces a new status "RunningFast", I don't care. virt-who needs to report mapping of VMs to nodes; it should not care about anything else.

Comment (pkliczewski) [reply virtwho/virt/kubevirt/kubevirt.py]: OK, I see your point. Will update this PR. Thanks!

**Comment by pkliczewski** [comment]
@wottop What do you think?

File: virt-who-config.5 [hunk 2]
```diff
@@ -145,6 +145,9 @@ Path to Kubernetes configuration file which contains authentication and connecti
 .TP
 \fB#kubeversion\fR
 API version used to override kubevirt api version fetched from the cluster. Used by kubevirt option
+.TP
+\fB#insecure\fR
+Allows not to validate tls certificates during connection to kubevirt
```
Comment (wottop) [inline virt-who-config.5]: Eliminate validation of tls certificates ...

**Review by wottop — CHANGES_REQUESTED** [review]
Just one suggested change in the man page

File: virt-who-config.5 (hunk 2 above)
Comment (pkliczewski) [reply virt-who-config.5]: Done

**Comment by wottop** [comment]
ack

**Pull request merged**